com.github.zocker_160.SyncThingy.yml

```diff
@@ -0,0 +1,65 @@
+app-id: com.github.zocker_160.SyncThingy
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+rename-icon: logo-512
+rename-desktop-file: SyncThingy.desktop
+command: SyncThingy
+
+finish-args:
+    - --share=ipc
+    - --share=network
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --device=dri
+    - --filesystem=host
+    - --talk-name=org.kde.StatusNotifierWatcher
+    - --talk-name=org.freedesktop.Notifications
+    - --talk-name=org.freedesktop.portal.Background
+
+modules:
+    - name: syncthing
+      buildsystem: simple
+      build-options:
+        env:
+          GOBIN: /app/bin
+      build-commands:
+        - |
+          . /usr/lib/sdk/golang/enable.sh
+          export GOPATH=$PWD/go
+          go version
+          env GOFLAGS=-mod=vendor go run build.go -no-upgrade build
+      post-install:
+        - install -D -m755 syncthing -t /app/bin
+        - install -D -m644 logo-512.png -t /app/share/icons/hicolor/512x512/apps
+      sources:
+        - generated-sources.json
+
+        - type: archive
+          url: https://github.com/syncthing/syncthing/releases/download/v1.20.2/syncthing-source-v1.20.2.tar.gz
+          sha256: 9e717e3e379682739e9cfb624307d0295d1ecb4dbb6a0a55dc102b79b23b353b
+          x-checker-data:
+            type: anitya
+            project-id: 11814
+            url-template: https://github.com/syncthing/syncthing/releases/download/v$version/syncthing-source-v$version.tar.gz
+
+        - type: file
+          url: https://raw.githubusercontent.com/syncthing/syncthing/9cc49aea7752a40f63011277bd1db4f85763b391/assets/logo-512.png
+          sha256: d0275a171c811209fa5d39b5ccdf7504fc356e40144cb98cb1d85e2acd4793e7
+              
+    - name: SyncThingy
+      buildsystem: cmake-ninja
+      post-install:
+        - install -D -m644 linux_packaging/SyncThingy.desktop -t /app/share/applications
+        - install -D -m644 linux_packaging/metainfo.xml -t /app/share/metainfo
```

Comment:
this clearly won't be found as it needs a `rename-appdata-file` or be installed with the right name

Reply:
yes....I am so stupid